### PR TITLE
Issue #SB-19153 feat: clean up the unwanted labels in portal

### PR DESCRIPTION
**Pull request by santoshilimi**

### CODE REVIEW

File: src/app/resourcebundles/data/consumption/en.properties [hunk 1]
```diff
@@ -1059,7 +967,6 @@ frmelmnts.lbl.specError = Password must contain at least 1 special character.
 frmelmnts.lbl.validEmail = Enter a valid email address
 messages.emsg.m0050 = Failed to validate OTP. Try again later.
 messages.imsg.m0086 = Incorrect OTP. Number of attempts remaining : {remainingAttempt}
-messages.imsg.m0087 = Not permitted to merge account
```
Comment (sudip-mukherjee) [inline src/app/resourcebundles/data/consumption/en.properties:1062]: check merge-account-status.component.html

File: src/app/resourcebundles/data/consumption/en.properties [hunk 2]
```diff
@@ -749,29 +719,10 @@ messages.stmsg.m0022 = Submit one of your drafts for review. Content is publishe
...
 messages.stmsg.m0083 = You have not shared content with any one yet
-messages.stmsg.m0088 = Enter a valid password
-messages.stmsg.m0089 = Enter a valid email address
-messages.stmsg.m0090 = Select languages
-messages.stmsg.m0091 = Enter a valid mobile number
-messages.stmsg.m0094 = Enter valid percentage
-messages.stmsg.m0095 = Your request is received successfully. The file will be sent to your registered email address shortly. Check your registered email address to view it.
-messages.stmsg.m0104 = Enter valid grade
 messages.stmsg.m0108 = Your Progress
-messages.stmsg.m0113 = Enter a valid start date
```
Comment (sudip-mukherjee) [inline src/app/resourcebundles/data/consumption/en.properties:774]: this is used in multiple html file: messages.stmsg.m0113 = Enter a valid start date
check create-course-batch, update-course-batch, create-batch component's HTML file

File: src/app/resourcebundles/data/consumption/en.properties [hunk 3]
```diff
@@ -691,26 +679,12 @@ messages.imsg.m0064 = delete the other account
 messages.imsg.m0065 = Account merge initiated successfully
 messages.imsg.m0066 = you will be notified once it is completed
 messages.imsg.m0086 = Incorrect OTP. Number of attempts remaining : {remainingAttempt}
-messages.imsg.m0087 = Account merge not permitted
-messages.imsg.m0067 = Could not initiate the account merge. Click the Merge Account option in the Profile menu and try again
```
Comment (sudip-mukherjee) [inline src/app/resourcebundles/data/consumption/en.properties:695]: messages.imsg.m0067 = Could not initiate the account merge. Click the Merge Account option in the Profile menu and try again
> In merge-account-status it is been used

File: src/app/resourcebundles/data/consumption/en.properties [hunk 4]
```diff
@@ -691,26 +679,12 @@ messages.imsg.m0064 = delete the other account
 messages.imsg.m0065 = Account merge initiated successfully
 messages.imsg.m0066 = you will be notified once it is completed
 messages.imsg.m0086 = Incorrect OTP. Number of attempts remaining : {remainingAttempt}
-messages.imsg.m0087 = Account merge not permitted
```
Comment (sudip-mukherjee) [inline src/app/resourcebundles/data/consumption/en.properties:694]: this is also in use: merge-account-status component

File: src/app/resourcebundles/data/consumption/en.properties [hunk 5]
```diff
@@ -652,21 +647,15 @@ messages.fmsg.m0080 = Failed to fetch the badge, try again later...
 messages.fmsg.m0082 = This course is not open for enrolment
 messages.fmsg.m0085 = There was a technical error. Try again.
 messages.fmsg.m0086 = This training is retired by the author and hence is not available anymore
-messages.fmsg.m0087 = Please wait
-messages.fmsg.m0088 = We are fetching details.
```
Comment (sudip-mukherjee) [inline src/app/resourcebundles/data/consumption/en.properties:656]: app -loader-component is using messages.fmsg.m0088

**Comment by santoshilimi** [comment]
raised another pr due change in base branch

**Pull request closed without merging**